packages/util/src/clone-element-with-ref.ts

```diff
@@ -13,7 +13,7 @@ export const cloneElementWithRef = <
   T extends HTMLElement
 >(
   el: ReactHTMLElement<T>,
-  outerRef: MutableRefObject<T>,
+  outerRef: MutableRefObject<T | undefined>,
```

Comment:
Most of the code we're righting has refs that match this type.  We rarely declare useRef with an initial value of type T, which is the only way we avoid undefined | null.